### ISSUE TITLE
Block comment in the middle of the code "disable" Sourcery

### ISSUE THREAD

**Issue by marcoaaguiar** [sourcery-ai/sourcery#98]
### Issue description or question
Block comment in the middle of the code "disable" Sourcery, example
```python 
class MyClass:
    def some_func(self):
        self.other_func()

        # comment
        # commented out code
        # even more comment

        # Comment
        if not self.L.numel() == 1:
            raise Exception()
 ```
It should suggest a De Morgan simplification in the if condition.

Things works properly if:
- the 3 lines of comment (LoC) are removed
- the empty line between the last LoC and the single of LoC before the `if` is removed

It still won't work if the single line of comment is removed.

### Sourcery Version
0.8.2

### Code editor or IDE name and version

Discovered on NeoVim 0.5.0

### OS name and version

Ubuntu 20.04 via WSL

**Comment by Hellebore** [comment]
Fix identified - will be in the next release.

**Comment by Hellebore** [comment]
Resolved with release of 0.8.3.
